vsix/check_installed.ps1

```diff
@@ -0,0 +1,42 @@
+
+
+if (${env:APPVEYOR_BUILD_WORKER_IMAGE} -eq "Visual Studio 2015")
+{
+    $devenv = "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\devenv.com"
+}
+else
+{
+    $visualStudioInstallation = & "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe" -latest -products * -requires Microsoft.VisualStudio.Component.VSSDK -property installationPath
+    $devenv = Join-Path $visualStudioInstallation 'Common7\IDE\devenv.com'
+}
+
+Write-Host "devenv: $devenv"
+
+$ntimes = 5
+For ($i=1; $i -le $ntimes; $i++) {  # Run 10 times
```

Comment:
do we still need 5 attempts?

Reply:
I prefer to remove it in another PR, just in case we need to recover that behavior, it will be easier to find.